CMakeLists.txt

```diff
@@ -958,6 +961,25 @@ if (BUILD_TESTING)
   math (EXPR CTEST_LONG_TIMEOUT "${DART_TESTING_TIMEOUT} * 2")
   math (EXPR CTEST_VERY_LONG_TIMEOUT "${DART_TESTING_TIMEOUT} * 3")
 
+  option (HDF5_TEST_API "Execute HDF5 API tests" OFF)
```

Comment:
You should be more explicit about this being for VOL developers (experts, advanced users), etc...